apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -86,11 +101,14 @@ $tokensTable-border-width: 1px;
     padding: 0;
     border: $tokensTable-border-width solid f.$color-border-default;
     border-radius: f.$radius-03;
+    background-color: f.$color-background-subtle;
 
     @include f.media('>=wide') {
         padding: f.spacing(e) 0;
         border-radius: 0;
         border-width: 0 0 $tokensTable-border-width 0;
+        background-color: f.$color-background-default;
+        min-height: 128px;
```

Comment:
I've definitely seen this number elsewhere, I think for the row heights - can it become a variable please? :)